frontend/src/components/TourDeFoos/TourDeFoos.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import {Button, createMuiTheme, MuiThemeProvider} from "@material-ui/core";
+import {Menu} from '../Menu';
+import {grey} from "@material-ui/core/colors";
+import {Centered} from "../Shared/Centered";
+import {TournamentRegistrations} from "../TournamentRegistrations";
+import {VerticalSpace} from "../Shared/VerticalSpace";
+import {APP_NAME} from "../../constants/constants";
+
+const theme = createMuiTheme({
+    palette: {
+        primary: {
+            main: grey[800],
+            contrastText: "#E3E152",
+        },
+    },
+});
+
+function TourDeFoos() {
+    const menuButtonsListProps = [
+        {
+            textName: "Turniej",
+            onClick: () => {
+                console.log("Tournament Button was clicked!")
+            },
+        },
+        {
+            textName: "Zawodnicy",
+            onClick: () => {
+                console.log("Tournament Button was clicked!")
```

Comment:
tu chyba fajnie zamiast tournament to Players button :-)

Reply:
:)